movai_core_shared/core/base_command.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC, abstractclassmethod, abstractmethod
+import sys
+
+from movai_core_shared.logger import Log
+
+class BaseCommand(ABC):
+    """Base Class for the various tools commands."""
+
+    def __init__(self, **kwargs) -> None:
+        self.log = Log.get_logger(__name__)
+        self.kwargs = kwargs
+
+    def safe_execute(self, **kwargs) -> None:
+        try:
+            self.execute(**kwargs)
+            sys.exit(0)
+        except Exception as e:
+            self.log.error(e)
```

Comment:
convert the exception to string **str(e)**

Reply:
done